Makefile

```diff
@@ -9,10 +9,32 @@ compile:
 
 # Installs dependencies
 .PHONY: install
-install:
+install: install-protoc
 	go mod vendor
 
 # Run all unit tests with coverage report
 .PHONY: unit
 unit:
 	go test -v -cover ./...
+
+.PHONY: install-protoc
+install-protoc:
+# Rely on homebrew in the case of MacOSX for now since CI is going to be running only Linux
+ifeq ($(shell uname -s), Darwin)
+	brew install protobuf
```

Comment:
can we provide an error message if brew is not installed?

Reply:
It'll be pretty clear that `brew` is not installed, i.e., in case `brew` is not installed you're going to see something like `make: breww: No such file or directory`